stanford-earth/stanford_news
# EARTH-151 EARTH-153 EARTH-154 Earth matters page

# (finally) READY FOR REVIEW

# Summary
- Pattern style option for the view to wrap the entire view in a pattern including header & footer stuff (still need to wrap in the filters)
- Custom path processor. This converts paths like `/earth-matters/34` into `/earth-matters/term-name`
- Custom parameter converter. this loads the term entity to pass to the page controller.
- Earth matters page with dynamic routes.

# Needed By (Date)
- launch

# Urgency
- high

# Steps to Test
1. composer update & sync.
2. Checkout this branch and `EARTH-154-earth-matters-views` branch of [stanford_components](https://github.com/SU-SWS/stanford_components/pull/86)
3. clear cache
4. `drush features-import stanford_news -y; drush en stanford_news_earth_matters -y`
5. go to /earth-matters page and make sure it works as designed. kinda like [this page](http://earthode43.prod.acquia-sites.com/earth-matters)

# Associated Issues and/or People
- https://github.com/SU-SWS/stanford_components/pull/86
- http://earthode43.prod.acquia-sites.com/earth-matters

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
